Source/Core/Form.swift

```diff
@@ -346,3 +346,17 @@ extension Form {
         kvoWrapper.sections.insert(section, at: formIndex == NSNotFound ? 0 : formIndex + 1 )
     }
 }
+
+extension Form {
+    
+    @discardableResult
+    public func validates() -> [ValidationError] {
```

Comment:
i added the annotation because we may want to run validations and discard the errors since `row.validationErrors` holds it. 

```
let errors: [ValidationError] = row.validationErrors
````